app/models/upscan/UpscanInitiateResponseModel.scala

```diff
@@ -0,0 +1,38 @@
+
+package models.upscan
+
+import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue, Json, Reads}
+
+case class UpscanInitiateResponseModel(
+                                   reference: Reference,
+                                   uploadRequest: UploadFormTemplateRequest
+                                 )
+case class Reference(value: String)
```

Comment:
I'm not sure in the need for a wrapper class for the file reference? Up to you if you want to keep it but it's just a String?

Reply:
Makes sense, will change to a string and can remove the format in the object